### DISCUSSION TITLE
Field extension

### DISCUSSION THREAD

**Discussion by PhaNtomBek** [sinamics/ztnet#305] [Ideas]
Good day.
Can this field be made a little larger?
I indicate a comment there, but it is not visible.
![image](https://github.com/sinamics/ztnet/assets/97723224/d24a891c-6ea2-4f09-a9c6-82900110ce2a)

**Comment by sinamics** [comment]
I moved this to an issue instead.
https://github.com/sinamics/ztnet/issues/306
